src/Core/Utilities/CoreHelpers.cs

```diff
@@ -182,26 +184,17 @@ public static Uri GetUri(string uriString)
 
         public static Dictionary<string, string> GetQueryParams(string urlString)
         {
-            var dict = new Dictionary<string, string>();
-            if (!Uri.TryCreate(urlString, UriKind.Absolute, out var uri) || string.IsNullOrWhiteSpace(uri.Query))
```

Comment:
I'd re-add this check again to check that the uri is valid and if not just early return an empty Dict